config/http.conf

```diff
@@ -1,18 +1,33 @@
+daemon off;
+
+worker_processes auto;
```

Comment:
Setting this to `auto` allows nginx to deremine the optimal number of processes based on number of cores.

Reply:
Don't we already do this in `main.conf`? 🤔 

https://github.com/Intellection/docker-nginx-proxy/blob/b87f59bfe036dff06870fd7afb854ebc22aac108/config/main.conf#L1

Reply:
Yeah, forgot about that. I've moved the configuration to there.